src/utils/db.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const { Sequelize } = require('sequelize');
+
+const sequelize = new Sequelize('postgres', 'postgres', '1111', {
```

Comment:
Consider using environment variables instead of hardcoded values for database credentials and host to increase security and flexibility.